woocommerce/woocommerce-admin
# Add the favorites tooltip to the navigation

Fixes #6213 

Adds a favorite tooltip on first time navigation to an extension category.

Note that this hides the tooltip when the nav is folded.  Expanding the nav may not trigger the tooltip again and a level change may be needed to retrigger the tooltip.  I think the workaround here might be moving the expanded state into the data store so we can unmount components, but a larger refactor will be needed to do this.

### Screenshots
<img width="662" alt="Screen Shot 2021-02-10 at 2 08 35 PM" src="https://user-images.githubusercontent.com/10561050/107559325-f2146500-6ba9-11eb-8144-c9566c96d96a.png">

### Detailed test instructions:

1. Delete the `woocommerce_navigation_favorites_tooltip_hidden` option if it exists.
2. Install and activate an extension that registers an category.
3. Navigate to that extension's category.
4. Note the tooltip.
5. Try folding the nav or resizing to a smaller screen and make sure it's not shown when the nav is folded.
5. Close the tooltip.
6. Refresh and navigate back to that category.
7. Make sure the tooltip doesn't appear again.
